File: Antibody.CareToKnowPro.CRM/Models/UserUnsubscribe.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Antibody.CareToKnowPro.CRM.Models
{
    public partial class UserUnsubscribe
    {
        [Key]
        public int UnsubscribeId { get; set; }
        public int UserId { get; set; }
        public int? ReasonId { get; set; }
        [Column("other")]
        [StringLength(200)]
        public string Other { get; set; }

        [ForeignKey("ReasonId")]
        [InverseProperty("UserUnsubscribe")]
        public virtual UnsubscribeReasons Reason { get; set; }
        [ForeignKey("UserId")]
        [InverseProperty("UserUnsubscribe")]
        public virtual User User { get; set; }
    }
}

File: Antibody.CareToKnowPro.CRM/ClientApp/src/app/model/userUnsubscribe.ts
import { IUnsubscribeReasons as UnsubscribeReasons } from './unsubscribeReasons';
import { IUser as User } from './user';

export interface IUserUnsubscribe { 
    unsubscribeId?: number;
    userId?: number;
    reasonId?: number;
    other?: string;
    reason?: UnsubscribeReasons;
    user?: User;
}
